src/services/event.js

```diff
@@ -273,11 +277,83 @@ const makeMatchesFilters = (filters) => (event) => {
   return false;
 };
 
+/**
+ * Splits up multiple occurrences of events into individual events
+ * and re-sorts the array
+ *
+ * @param {Object[]} events the array of events to process
+ * @returns {Object[]} the flattened array of events after splitting multiple occurrences
+*/
+const processMultipleOccurences = (events) => {
+
+  var splitEvents = [];
```

Comment:
You used `var` here.
```suggestion
  const splitEvents = [];
```